### DISCUSSION TITLE
文化幣

### DISCUSSION THREAD

**Discussion by seastar0687** [1101-GS4719-C/Python-Community-GS4719-C#3700] [General]
文化幣的使用範圍大部分都圍繞在文藝活動上面，對於我這種對文藝活動比較沒興趣的人說其實不好花
雖然說文化幣可以買小說，漫畫，甚至成人漫畫跟同人誌。但平常沒有收藏習慣的人，也不太會因為文化幣買一套書回家
所以113年的文化幣我直到跨年前幾天都還沒花半分錢，好不容易花完，114年的文化幣又要來了
大家都怎麼使用文化幣呢，有甚麼推薦的地方嗎

**Comment by poyen0806** [comment]
今天原本去西門打算買幾本小說回來囤著看，結果逛了半天沒找到想看的。最後就隨手買了個滑鼠墊備用，剩下的文化幣還幫同學買了些周邊商品。

其實有想買關於 Python 的 NLTK 或一些演算法的教學書，不過地方太遠了，下次一定。

啊結果回來後沒多久，就要準備領114年的文化幣了。

**Comment by SendaiHatsuki** [comment]
我的文化幣都直接拿來買上課要用的原文書，平常一本快2000的書可以用文化幣買，感覺沒那麼心疼。如果要發揮文化幣最的作用建議去獨立書店買書，納編會有很多優惠，可以打折之類的。

**Comment by lichilun** [comment]
我推薦（大學城書局），在它這邊消費文化幣1200元，可以反饋600元，對需要大量文具的我是個好選擇，因此這兩年文化幣一發下來，我都會去大學城書局購買：原文書、計算機、文具、廣告紙（美宣工人需要），真的超方便的！！！

**Comment by BOSENCHANG2004** [comment]
我並不常看書，但我之前比較常拿文化去去買自己喜歡動畫的繪本，我認為這非常的有收藏價值是非常划算的交易。

**Comment by a414414** [comment]
買韓團專輯

**Comment by hollowha** [comment]
可以拿去作手作，做香水

**Comment by kai1660** [comment]
文化幣可以去買書，平常都沒有買實體書的習慣，可以藉文化幣的活動，來買你喜歡的書，也有很多其他活動，例如手做之類的課程，應該也都可以使用文化幣去體驗，希望大家都可以好好利用文化幣。

**Comment by RoyLin1014** [comment]
我覺得跟朋友玩密室逃脫是不錯的選擇，有些兩人就可以成團，很好玩

**Comment by Twicky123** [comment]
文化幣對我來說就是可以拿來買書的好工具，也可以拿去看國產電影，要是真的沒有用途的話，其實也有社群可以交易成現金，但這塊就要小心防範詐騙!

**Comment by 77JoyWang77** [comment]
密室逃脫，假日約朋友去玩

**Comment by shrshu38** [comment]
我直接去音樂會用爆，平常捨不得買的1200 1600門票都半價到免費。文化幣出來了才第一次到衛武營聽到真的管風琴，到國家音樂廳聽協奏曲。

**Comment by lilipop6405** [comment]
我去年文化幣是最後一個月我同學突然問我文化幣怎麼花我才發現還有1200...，所以我最後沒時間只好去敦煌買書。到現在那本小書才看一點點，而另外一本工具書是有用到拉，但是用的不多，才用過三四次。我覺得我浪費了許多資源。題外話，我已經變老人了，今年沒有文化幣。

**Comment by yurukung** [comment]
我會拿去當作歷史博物館門票、美術館門票、買書、買專輯、買音樂祭門票，蠻好用的。

**Comment by ZorasWu** [comment]
我都會去獨立書店花 因為1200可以變成1800 我就可以買很多我需要或喜歡的書 但是他現在倒了 我正在尋找哪家獨立書店有我愛看的書又符合我的需求的

**Comment by alice20040127** [comment]
今年的文化幣我拿去做手作戒指了，感覺是一項蠻值得體驗的活動，第一次嘗試做金工感覺非常緊張，練習刻字的時候明明都能刻的很整齊很漂亮，但是到了真正刻在戒指上時，可能就有一種-只能操作一次沒有機會失敗的緊張及壓力在，感覺完全是沒有發揮出先前練習時的一分一毫，刻的那是真的叫歪七扭八啊。

**Comment by wuyu06** [comment]
可以考慮用來支持小型藝文展覽、戲劇或音樂會，體驗不同文化形式 也可以看看附近有沒有特別的藝文店鋪或活動能用文化幣

**Comment by edawdewv** [comment]
去年我就用文化幣去玩了一次密室逃脫，當時是跟朋友一起去的。進入密室的那一刻，完全沉浸在故事情境中，解謎過程既刺激又有趣。用文化幣支付讓我覺得很划算，因為除了娛樂，也算是一種文化體驗。這種互動式的遊戲讓人感受到創意和想像力的魅力，跟單純看書或聽音樂不太一樣，非常值得嘗試。

**Comment by XiangLuen** [comment]
之前都拿去買小說，因為其他展覽感覺都很無聊，不如去買些喜歡的小說，還能抽文化幣，感覺就很棒

**Comment by Wei0810** [comment]
今年是我最後一年拿文化幣了，我通常都會去獨立書店花，可以順便湊加碼活動XD 之前有一年加上抽獎拿到了兩千多點，而且獨立書店也能找到一些好看的書籍，我通常會順便再書店逛一下午，拓展自己眼界
